metals/src/main/scala/scala/meta/internal/metals/Messages.scala

```diff
@@ -480,6 +499,13 @@ object Messages {
         s"Class '$cls' not found."
       )
     }
+
+    def show(msg: String): MessageParams = {
```

Comment:
```suggestion
    def errorMessageParams(msg: String): MessageParams = {
```
?

Reply:
Ah yea, this was left over from me wanting a "generic" pass-through way to just show an error message where the full message was provided. I've gone ahead and moved this up a level and converted it to a generic `showError` method.